{
  "info": {
    "author": "Zope Corporation and Contributors",
    "author_email": "zope-dev@zope.org",
    "bugtrack_url": null,
    "classifiers": [
      "Development Status :: 5 - Production/Stable",
      "Environment :: Web Environment",
      "Framework :: Zope3",
      "Intended Audience :: Developers",
      "License :: OSI Approved :: Zope Public License",
      "Natural Language :: English",
      "Operating System :: OS Independent",
      "Programming Language :: Python",
      "Programming Language :: Python :: 2",
      "Programming Language :: Python :: 2.7",
      "Programming Language :: Python :: 3",
      "Programming Language :: Python :: 3.3",
      "Programming Language :: Python :: 3.4",
      "Programming Language :: Python :: 3.5",
      "Programming Language :: Python :: 3.6",
      "Programming Language :: Python :: Implementation :: CPython",
      "Programming Language :: Python :: Implementation :: PyPy",
      "Topic :: Internet :: WWW/HTTP"
    ],
    "description": "==============\nzope.errorview\n==============\n\nProvides basic HTTP and Browser views for common exceptions.\n\nRefactored from `zope.app.http`_.exception and `zope.app.exception`_.\n\n.. _`zope.app.http`: http://pypi.python.org/pypi/zope.app.http\n.. _`zope.app.exception`: http://pypi.python.org/pypi/zope.app.exception\n\n\n=========\n CHANGES\n=========\n\n1.1 (2018-01-10)\n================\n\n- Additional fixes for Python 3 compatibility.\n\n  NOTE: The error view base classes now set a Content-Type response header to\n  \"text/plain\". If your error view subclassing from the zope.errorview classes\n  return a response body other than \"text/plain\" you need to explicitly set\n  the Content-Type in your views.\n\n1.0.0 (2017-05-10)\n==================\n\n- Add support for Python 3.4, 3.5, 3.6 and PyPy.\n\n- Fix typo in Dutch translation\n\n0.11 (2011-06-28)\n=================\n\n- Added nl translations.\n\n0.10 (2011-02-08)\n=================\n\n- Exception views do not by default provide ISystemErrorView anymore as it\n  would result in duplicate log output. The mixin class still exists for\n  writing custom error views that do provide ISystemErrorView.\n\n0.9 (2011-01-20)\n================\n\n- Initial release.\n\n\n",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": 0,
      "last_month": 0,
      "last_week": 0
    },
    "home_page": "http://github.com/zopefoundation/zope.errorview",
    "keywords": "zope3 http error view",
    "license": "ZPL 2.1",
    "maintainer": "",
    "maintainer_email": "",
    "name": "zope.errorview",
    "platform": "",
    "project_url": "https://pypi.org/project/zope.errorview/",
    "release_url": "https://pypi.org/project/zope.errorview/1.1/",
    "requires_dist": [
      "zope.testrunner; extra == 'test'",
      "zope.testing; extra == 'test'",
      "zope.browserpage; extra == 'test'",
      "zope.browser; extra == 'test'",
      "zope.authentication; extra == 'test'",
      "zope.browserpage; extra == 'browser'",
      "zope.browser; extra == 'browser'",
      "zope.authentication; extra == 'browser'",
      "zope.security",
      "zope.publisher",
      "zope.interface",
      "zope.i18nmessageid",
      "zope.i18n",
      "zope.component",
      "setuptools"
    ],
    "requires_python": "",
    "summary": "Basic HTTP and Browser exception views.",
    "version": "1.1"
  },
  "releases": {
    "0.10": [
      {
        "comment_text": "",
        "digests": {
          "md5": "d5eabcc779f1bed5fc7d4b430ea94549",
          "sha256": "b715021c1fb8a20c1f4a604dab70d23f256351d493ec07335750c7632affca63"
        },
        "downloads": -1,
        "filename": "zope.errorview-0.10.tar.gz",
        "has_sig": false,
        "md5_digest": "d5eabcc779f1bed5fc7d4b430ea94549",
        "packagetype": "sdist",
        "python_version": "source",
        "size": 7588,
        "upload_time": "2011-02-08T11:45:05",
        "url": "https://files.pythonhosted.org/packages/b7/eb/a9715f871cc1344f31d499e924aa60c2873d6aacddfba0e504f1b11135a0/zope.errorview-0.10.tar.gz"
      }
    ],
    "0.11": [
      {
        "comment_text": "",
        "digests": {
          "md5": "7ad7e5404557bd7cfbd9126b7f2c067a",
          "sha256": "012e10014888dda5945cf3ab76ad5c67328003c7f3e72109ce9e0288600bf75d"
        },
        "downloads": -1,
        "filename": "zope.errorview-0.11.tar.gz",
        "has_sig": false,
        "md5_digest": "7ad7e5404557bd7cfbd9126b7f2c067a",
        "packagetype": "sdist",
        "python_version": "source",
        "size": 8424,
        "upload_time": "2011-06-28T15:16:30",
        "url": "https://files.pythonhosted.org/packages/be/6a/96fa7add70839bc6f699a9d2b21e2c87a4240026fb3dd201fec40dd54bb5/zope.errorview-0.11.tar.gz"
      }
    ],
    "0.9": [
      {
        "comment_text": "",
        "digests": {
          "md5": "6ed305a5ba3e75798913b819e0466839",
          "sha256": "766724f4579cca102b46eb94fc5097931bc073745a91c7ffc869062d57dcf188"
        },
        "downloads": -1,
        "filename": "zope.errorview-0.9.tar.gz",
        "has_sig": false,
        "md5_digest": "6ed305a5ba3e75798913b819e0466839",
        "packagetype": "sdist",
        "python_version": "source",
        "size": 7249,
        "upload_time": "2011-01-20T12:42:45",
        "url": "https://files.pythonhosted.org/packages/7e/fe/2b35d59d10d8b446659f726cc8ec4378dd77c7529491062718b0a388ad2c/zope.errorview-0.9.tar.gz"
      }
    ],
    "1.0.0": [
      {
        "comment_text": "",
        "digests": {
          "md5": "0518273306ec387cb97aef8ba21fb874",
          "sha256": "1a872a1f35c2b0898ae913de78e72c48dd703a349f78f3b204331a6d55017a83"
        },
        "downloads": -1,
        "filename": "zope.errorview-1.0.0-py2.py3-none-any.whl",
        "has_sig": false,
        "md5_digest": "0518273306ec387cb97aef8ba21fb874",
        "packagetype": "bdist_wheel",
        "python_version": "py2.py3",
        "size": 15582,
        "upload_time": "2017-05-10T11:14:40",
        "url": "https://files.pythonhosted.org/packages/25/55/8d4f69a6d3e9c9601fa6876000351d6d3606e4f7dbe762bf948e76fd2640/zope.errorview-1.0.0-py2.py3-none-any.whl"
      },
      {
        "comment_text": "",
        "digests": {
          "md5": "417f2b7a2484058e92b5687346e87f9a",
          "sha256": "f5b63405386210f183c7e65d815aee24a826ffcd81c7bf47e25f6f6fb81e7799"
        },
        "downloads": -1,
        "filename": "zope.errorview-1.0.0.tar.gz",
        "has_sig": false,
        "md5_digest": "417f2b7a2484058e92b5687346e87f9a",
        "packagetype": "sdist",
        "python_version": "source",
        "size": 11028,
        "upload_time": "2017-05-10T11:14:41",
        "url": "https://files.pythonhosted.org/packages/ec/d9/9b7e482302e4c533a1b2bf9f90b11d5301b05e167ff698bd8f3732438ac3/zope.errorview-1.0.0.tar.gz"
      }
    ],
    "1.1": [
      {
        "comment_text": "",
        "digests": {
          "md5": "d97b6de3dd7fec748f5e6646d9fe4f95",
          "sha256": "a3a044147ef7ce981d84b5d72ef03d7a40249a61da99800c95112cd32bd33620"
        },
        "downloads": -1,
        "filename": "zope.errorview-1.1-py2.py3-none-any.whl",
        "has_sig": false,
        "md5_digest": "d97b6de3dd7fec748f5e6646d9fe4f95",
        "packagetype": "bdist_wheel",
        "python_version": "py2.py3",
        "size": 15948,
        "upload_time": "2018-01-10T10:02:24",
        "url": "https://files.pythonhosted.org/packages/ef/4a/5e354e2351466f02667772cad8c509623e07ccc72fe4eb9b6d277468c696/zope.errorview-1.1-py2.py3-none-any.whl"
      },
      {
        "comment_text": "",
        "digests": {
          "md5": "dd0b50c970f5216cf9d2f5a9c171fd4c",
          "sha256": "dd77fb84bfe1956ed7e7e00ba98e338efa3dd5354b219d37e1cc9944fb3087f4"
        },
        "downloads": -1,
        "filename": "zope.errorview-1.1.tar.gz",
        "has_sig": false,
        "md5_digest": "dd0b50c970f5216cf9d2f5a9c171fd4c",
        "packagetype": "sdist",
        "python_version": "source",
        "size": 11431,
        "upload_time": "2018-01-10T10:02:25",
        "url": "https://files.pythonhosted.org/packages/da/56/0a8bb0e2e42ce900d6a71029884b02c1e5c4d85590f85f9ab2a228197510/zope.errorview-1.1.tar.gz"
      }
    ]
  },
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "md5": "d97b6de3dd7fec748f5e6646d9fe4f95",
        "sha256": "a3a044147ef7ce981d84b5d72ef03d7a40249a61da99800c95112cd32bd33620"
      },
      "downloads": -1,
      "filename": "zope.errorview-1.1-py2.py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "d97b6de3dd7fec748f5e6646d9fe4f95",
      "packagetype": "bdist_wheel",
      "python_version": "py2.py3",
      "size": 15948,
      "upload_time": "2018-01-10T10:02:24",
      "url": "https://files.pythonhosted.org/packages/ef/4a/5e354e2351466f02667772cad8c509623e07ccc72fe4eb9b6d277468c696/zope.errorview-1.1-py2.py3-none-any.whl"
    },
    {
      "comment_text": "",
      "digests": {
        "md5": "dd0b50c970f5216cf9d2f5a9c171fd4c",
        "sha256": "dd77fb84bfe1956ed7e7e00ba98e338efa3dd5354b219d37e1cc9944fb3087f4"
      },
      "downloads": -1,
      "filename": "zope.errorview-1.1.tar.gz",
      "has_sig": false,
      "md5_digest": "dd0b50c970f5216cf9d2f5a9c171fd4c",
      "packagetype": "sdist",
      "python_version": "source",
      "size": 11431,
      "upload_time": "2018-01-10T10:02:25",
      "url": "https://files.pythonhosted.org/packages/da/56/0a8bb0e2e42ce900d6a71029884b02c1e5c4d85590f85f9ab2a228197510/zope.errorview-1.1.tar.gz"
    }
  ]
}